src/cr/cube/cubepart.py

```diff
@@ -1153,24 +1153,6 @@ def table_margin(self):
         """
         return self._assembler.table_margin
 
-    @lazyproperty
-    def table_margin_unpruned(self):
```

Comment:
@scanny - In the other PR (https://github.com/Crunch-io/crunch-cube/pull/318#discussion_r649500157) you suggested removing this. Just making sure we're on the same page, this was an external interface presented on the `_Slice`, and will require changing the exporter. Does this feel worth it?

(The `unpruned_table_margin_range` below will allow us to get the same thing that we actually used in exporter)

Reply:
My understanding is that `.table_margin_unpruned` is only used by `exporter` to compute "table-margin-range" when it has determined that the table-margin is not a scalar, probably by seeing that the rows-dimension or columns-dimension is MR.

The table-margin-range value is a legitimate "total" and if exporter needs it (which it does) then `cr.cube` should provide it rather than leaving exporter and other clients to do that for themselves.

If we're going to fix that responsibility-leakage then exporter will have to lose some code and get that value from `_Slice.table_margin_range` or whatever instead. Once that's done the only call(s) to `.table_margin_unpruned` will be gone and it can disappear. There should be no legitimate need for an unpruned anything outside of `cr.cube`.

As far as timing is concerned, when you do that part is up to you, but it's an important part of measure consolidation in my view because that margin-range measure/total is now computed in `exporter` and needs to be consolidated into `cr.cube` so other clients (like SOA and maybe `rcrunch`) don't have to reimplement it.